src/main/java/org/broadinstitute/hellbender/tools/spark/transforms/BaseRecalibratorSparkFn.java

```diff
@@ -48,4 +53,36 @@ public static RecalibrationReport apply( final JavaPairRDD<GATKRead, ReadContext
         final StandardCovariateList covariates = new StandardCovariateList(recalArgs, header);
         return RecalUtils.createRecalibrationReport(recalArgs.generateReportTable(covariates.covariateNames()), quantizationInfo.generateReportTable(), RecalUtils.generateReportTables(combinedTables, covariates));
     }
+
+    /**
+     * Run the {@link BaseRecalibrationEngine} on reads and overlapping variants.
```

Comment:
Given that the tool and its execution in reads pipeline spark have been updated to use this method, we should just delete the other apply() function.